scripts/install-systemd-multi-user.sh

```diff
@@ -97,7 +97,7 @@ poly_configure_nix_daemon_service() {
         task "Setting up the nix-daemon systemd service"
 
         _sudo "to create the nix-daemon tmpfiles config" \
-              ln -sfn "/nix/var/nix/profiles/default$TMPFILES_SRC" "$TMPFILES_DEST"
+              mkdir -p "$(dirname "$TMPFILES_DEST")" && ln -sfn "/nix/var/nix/profiles/default$TMPFILES_SRC" "$TMPFILES_DEST"
```

Comment:
I don't think && will work with `_sudo` here. You would need to put this on a seperate line.